zerver/lib/outgoing_webhook.py

```diff
@@ -210,7 +210,10 @@ def fail_with_message(event: Dict[str, Any], failure_message: str) -> None:
     message_info = event["message"]
     content = "Failure! " + failure_message
     response_data = dict(content=content)
-    send_response_message(bot_id=bot_id, message_info=message_info, response_data=response_data)
+    try:
+        send_response_message(bot_id=bot_id, message_info=message_info, response_data=response_data)
+    except JsonableError:
+        pass
```

Comment:
The original issue report suggested doing this in `send_response_message`; how did you think about that decision?  I'm not sure what's correct.

I do note that `notify_bot_owner` likely is vulnerable to the same bug if the bot owner is deactivated; it looks like `process_success_response` already catches JsonableError (And then calls `send_response_message`).

So maybe we just need this in both `notify_bot_owner` and here; with a test for the other case.